app/Filament/Resources/TimeScheduleResource.php

```diff
@@ -0,0 +1,153 @@
+<?php
+
+namespace App\Filament\Resources;
+
+use App\Filament\Resources\TimeScheduleResource\Pages;
+use App\Filament\Resources\TimeScheduleResource\RelationManagers;
+use App\Models\TimeSchedule;
+use App\Models\User;
+use Carbon\Carbon;
+use Filament\Forms;
+use Filament\Forms\Components\Section;
+use Filament\Forms\Components\Select;
+use Filament\Forms\Components\TimePicker;
+use Filament\Forms\Form;
+use Filament\Forms\Get;
+use Filament\Resources\Resource;
+use Filament\Tables;
+use Filament\Tables\Columns\TextColumn;
+use Filament\Tables\Table;
+use Illuminate\Database\Eloquent\SoftDeletingScope;
+use Illuminate\Support\Collection;
+use Illuminate\Support\Facades\DB;
+
+class TimeScheduleResource extends Resource
+{
+    protected static ?string $model = TimeSchedule::class;
+
+    protected static ?string $navigationIcon = 'heroicon-o-rectangle-stack';
+
+    public static function form(Form $form): Form
+    {
+        return $form
+            ->schema([
+                Section::make('')
+                    ->schema([
+                        Select::make('role_id')
+                            ->label('Schedule For')
+                            ->options(DB::table('roles')->where('name', '!=', User::ROLE_ADMIN)->pluck('name', 'id'))
```

Comment:
@kolanaveen Please exclude 'Patient' role as well. Because time scheduling is only for people who is working in hospital.